Makefile

```diff
@@ -466,7 +466,8 @@ XGETTEXT_BIN=xgettext
 
 pot:
 	mkdir -p $(TEMPLATE_DIR)
-	$(XGETTEXT_BIN) --from-code=utf-8 --keyword=C_:1c,2 \
+	$(XGETTEXT_BIN) --from-code=utf-8 \
+		--keyword=C_:1c,2 --keyword=N_:1,2 \
```

Comment:
For `npgettext`, given that we're using `C_` for `pgettext`, logically that should be `NC_`. I didn't really test that one yet but other than the lacking npgettext method it should be working.

Reply:
But `CN_` would state the order: first context, then the next thing will be N'ed :)
(if we don't need to care about the involved gettext function names - but you decide)

Reply:
For reference, GTK also has an NC_ (although it means something completely different): https://developer.gnome.org/glib/stable/glib-I18N.html#NC-:CAPS

Anyway, I don't even know if we need `npgettext`. It's just with all the groundwork already there it seems silly not to add the method.